packages/jest-reporters/src/getSummary.ts

```diff
@@ -114,6 +114,11 @@ export default function getSummary(
   const testsTotal = aggregatedResults.numTotalTests;
   const width = (options && options.width) || 0;
 
+  const seed = `${
```

Comment:
I don't think we should print the seed value by default (as most people won't ever be needing it).

I think we should add anothing flag to config (`printSeedValue: boolean` or something) which defaults to `false`. But if a _manual_ `seed` value is provided through the CLI, it is set to `true` automatically.

That way if you do use the value in your tests, you can configure jest to print it. But people who don't use it, avoids extra noise in their output